package-lock.json

```diff
@@ -1,198 +1,164 @@
 {
   "name": "sandahl",
   "version": "0.0.0",
-  "lockfileVersion": 3,
+  "lockfileVersion": 1,
```

Comment:
@AlgotE It looks like the dependencies were installed with npm <= v6 indicated by the lockfileVersion. What node.js version do you have installed? I think we should try to stick to node.js 18 LTS, eslint seems to fail due to faulty dependencies.

Reply:
I will update node.js to 18 and re-add the dependencies